### PR TITLE
Added the stubs for template enhancements.

### PR DESCRIPTION
**Pull request by kbidarkar**

### CODE REVIEW

**Comment by kbidarkar** [comment]
My bad, will fix this soon.

**Comment by sghai** [comment]
ACK

File: tests/foreman/ui/test_template.py [hunk 1]
```diff
@@ -407,3 +407,108 @@ def test_positive_advanced_search(self):
...
+    @tier1
+    def test_positive_export_template_disable_safe_render(self):
+        """Assure templates can be exported after disabling safe render.
+
+        :id: c4c05c13-1ea3-4963-b2dc-ec826b77c9cb
+
+        :Steps:
+            1. Go to Provisioning template UI
+            2. Set Safe mode Rendering to No, from settings.
+            3. The template is exported successfully.
```
Comment (jyejare) [inline tests/foreman/ui/test_template.py]: Does this step need to modify or delete completely ?

Comment (kbidarkar) [inline tests/foreman/ui/test_template.py]: We just need to export after disabling the "safe mode rendering" from settings menu.
No modify or delete would be required.

**Pull request merged**